GliaWidgets/Sources/View/Chat/ChatStyle.swift

```diff
@@ -118,51 +118,45 @@ public class ChatStyle: EngagementStyle {
         assetsBuilder: RemoteConfiguration.AssetsBuilder
     ) {
         header.apply(
-            configuration: configuration?.header,
-            assetsBuilder: assetsBuilder
+            configuration: configuration?.header, assetsBuilder: assetsBuilder
         )
         connect.apply(
-            configuration: configuration?.connect,
-            assetsBuilder: assetsBuilder
+            configuration: configuration?.connect, assetsBuilder: assetsBuilder
         )
         visitorMessage.apply(
-            configuration: configuration?.visitorMessage,
-            assetsBuilder: assetsBuilder
+            configuration: configuration?.visitorMessage, assetsBuilder: assetsBuilder
         )
         operatorMessage.apply(
-            configuration: configuration?.operatorMessage,
-            assetsBuilder: assetsBuilder
+            configuration: configuration?.operatorMessage, assetsBuilder: assetsBuilder
         )
         messageEntry.apply(
-            configuration: configuration?.input,
-            assetsBuilder: assetsBuilder
+            configuration: configuration?.input, assetsBuilder: assetsBuilder
         )
         choiceCard.apply(
-            configuration: configuration?.responseCard,
-            assetsBuilder: assetsBuilder
+            configuration: configuration?.responseCard, assetsBuilder: assetsBuilder
         )
         audioUpgrade.apply(
-            configuration: configuration?.audioUpgrade,
-            assetsBuilder: assetsBuilder
+            configuration: configuration?.audioUpgrade, assetsBuilder: assetsBuilder
         )
         videoUpgrade.apply(
-            configuration: configuration?.videoUpgrade,
-            assetsBuilder: assetsBuilder
+            configuration: configuration?.videoUpgrade, assetsBuilder: assetsBuilder
         )
         callBubble.apply(
-            configuration: configuration?.bubble,
-            assetsBuilder: assetsBuilder
+            configuration: configuration?.bubble, assetsBuilder: assetsBuilder
         )
         pickMedia.apply(
-            configuration: configuration?.attachmentSourceList,
-            assetsBuilder: assetsBuilder
+            configuration: configuration?.attachmentSourceList, assetsBuilder: assetsBuilder
         )
         unreadMessageIndicator.apply(
-            configuration: configuration?.unreadIndicator,
-            assetsBuilder: assetsBuilder
+            configuration: configuration?.unreadIndicator, assetsBuilder: assetsBuilder
         )
         operatorTypingIndicator.apply(configuration: configuration?.typingIndicator)
 
+        unreadMessageDivider.apply(
+            lineColor: configuration?.newMessagesDividerColor, text: configuration?.newMessagesDividerText,
```

Comment:
I guess the `text` can also be on its own line

Reply:
I did it because of SwiftLint warning

Reply:
fixed